server/src/main/java/com/flipkart/varadhi/VaradhiApplication.java

```diff
@@ -95,7 +95,7 @@ private static Future<Vertx> createClusteredVertx(
         int port = 0;
         JsonObject memberInfoJson = new JsonObject(JsonMapper.jsonSerialize(memberInfo));
         EventBusOptions eventBusOptions = new EventBusOptions()
-                .setHost(memberInfo.hostname())
+                .setHost(memberInfo.host())
```

Comment:
EventBus has an option to set clusterpublichost. May be we can use host to be the podname and use ip to set publichost. Need to see node will get registered (via ip or name) in that case ?

Reply:
made the change. Now relying on clusterpublichost.